description.md

```diff
@@ -17,6 +17,121 @@ Each section details how a specific type of content is to be developed, stored a
+
+#### Purpose
+
+Quizzes provide a means to continuously evaluate students.
+This creates a positive feedback loop between trainers and trainees whereby both can track the students' progress in real time.
+
+Furthermore, quizzes serve as practice items for students to solve at their own pace.
+They may also be used for evaluation items that contribute to the students' overall grade.
+
+Another desired aspect is that quizzes be easily extractable and convertible between formats, as described [below](#deployment), and repurposed by other trainers.
+
+Despite all of their strengths, quizzes also come with a few shortcomings.
+One such downside is the binary nature of the feedback loop.
+Because multiple-choice answers may be either correct or wrong, there is little room for nuance regarding the feedback that students receive upon answering incorrectly.
+Similarly, quizzes cannot be easily tailored and personalized for the individual needs of each student.
+In addition, non-technical fields of study may find little benefit in quizzes as they rely more on arguments essays rather than concrete, technical information.
+
+#### Deployment
+
+Each trainer uses a specific teaching rendering platform.
+Most platforms support quizzes in various formats, such as whether they are graded or ungraded, whether the answers are multiple choice or text boxes, etc.
+Out of the existing teaching platforms, a few highlights are:
+
+- [Moodle](https://moodle.org/) is a free open-source learning platform dedicated to course management.
+It supports graded assignments, quizzes, storing teaching materials and many other educational activities.
+
+- [SmileyTutor (formerly called TutorWeb)]((https://tutor-web.net/)) is another free learning platform whose highlight is that it was built around the [SmyleyCoin cryptocurrency](#cryptocurrency-rewards) as a reward for completing drills and quizzes.
+
+- [tech.io](https://tech.io/) is another quiz platform.
+More simplistic than Moodle or SmileyTutor, it allows teachers to easily create _playgrounds_, which are learning environments designed around specific subjects, made up of reading material, followed by practical exercises and quizzes.
+
+Each of these platforms supports a different syntax for uploading questions.
+To make our quizzes comply with educational platforms easily, we need to store them in a generic format that is easily representable and automatically convertible to other required formats.
+For this reason and for the sake of consistency, we are storing questions in the same Markdown format as all of our written material.
+
+We are developing a modular and easily extendible converter that ports quizzes from Markdown to other formats.
+The format that we use to store quizzes is easily convertible to other formats.
+Therefore, we do not implement converters for all the existing formats.
+Instead, we do provide a program that is easily extendible to support any kind of format.
```

Comment:
This is not clear. What do we provide and what we don't provide?

Reply:
I think that an image may help here.